live-examples/js-examples/promise-constructor.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js" class="language-js">var p = new Promise(function(resolve, reject) {
```

Comment:
Nit: Not sure what the norm is with regards to examples on MDN but, I always feel like we should use variable names one would use in general. So here `var p` seems to not fit the bill. I would probably have done `var promise = ...` Thoughts @maboa @wbamberg

Reply:
Yes - that sounds reasonable Schalk. I always worry that it becomes slightly confusing / less readable having both promise and Promise floating about and for some reason I'm no longer a fan of the myPromise = new Promise ... type convention.

Reply:
Never ask a writer for their opinions on naming :).

I agree that `p` is not good but I don't like `promise` either. Since this is a timer, why not `timer`?

Reply:
In the end I went with promise1, promise2 etc. But I'm happy to change.

Reply:
I'm fine with that :).